cli/transaction/interfaces.py

```diff
@@ -0,0 +1,33 @@
+"""
+Provide implementation of the transaction interfaces.
+"""
+
+
+class TransactionInterfaces:
+    """
+    Implements transaction interface.
+    """
+
+    async def get_list(self, query):
+        """
+        Get list transaction by: ids, start, head, limit, reverse.
+
+        Arguments:
+            query (dict, optional): dictionary with specific parameters
+
+        Returns:
+            List of transaction.
```

Comment:
1. Remove `Returns` block because of summary line very clear.
2. Move summary line to:

```
Get a list of transactions.

List of transactions can be filtered by identifiers, start pagging, identifier of block's head...

Arguments:
   ...
```

The same for another method in interface and implementation.

Reply:
@dmytrostriletskyi, done.